utils/lists.ts

```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Randomized adventurer meals",
     url: "https://etherscan.io/address/0x615a610649e656485d9baf0ebe525496d7b78e24",
   },
+  {
+    name: "Cangjie",
+    description: "Cangjie(仓颉) was the creator of Chinese characters. The Cangjie NFTs are Chinese characters generated with Taiji, Wuxing, Yi Jing, and more.",
```

Comment:
Can you please make the description shorter? so that it fits in one line like others

Reply:
I have fixed the description. Please re-check it. Thank you.